ocs_ci/helpers/dr_helpers_ui.py

```
@@ -28,6 +32,665 @@
+    assert all(app_checks), (
+        f"Applications sidebar rows do not match expected apps: {protected_apps}"
+    )
+    acm_obj.take_screenshot()
+    log.info(
+        "Applications sidebar validated for apps: "
+        f"{[app['name'] for app in protected_apps]}"
+    )
+
+
+def wait_for_dr_topology_workloads_healthy(
+    workloads, scheduling_interval, pvc_interface=constants.CEPHBLOCKPOOL
```

Ruff (0.15.18)

[warning] Unused function argument: `pvc_interface`

(ARG001)